<!DOCTYPE html>
<html>
    <head>
        <title>miRNA Analysis from omiRas</title>
        <meta charset="utf-8">
        <link rel="stylesheet" type="text/css" media="screen" href="http://netdna.bootstrapcdn.com/twitter-bootstrap/2.1.0/css/bootstrap-combined.min.css">
        <link rel="stylesheet" type="text/css" media="screen" href="../../stylesheets/pygment_trac.css">
        <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/1.8.0/jquery-1.8.0.min.js" type="text/javascript"></script>
        <script src="http://netdna.bootstrapcdn.com/twitter-bootstrap/2.1.0/js/bootstrap.min.js" type="text/javascript"></script>
        <style>
            #main-body{
                margin-top: 40px;
			}
			.article_menu{
				margin: 10px 0 10px 5px;
				padding: 5px 0 10px 0;
				border: 1px solid #c4c4c4;
				box-shadow:0 0 10px #c4c4c4;
			}
			.article_menu>.s1{
				margin-left: 10px;
				margin-right:10px;
				padding: 5px 5px 5px;
				border-bottom: 1px solid silver;
			}
			.article_menu>.s2{
				margin-left: 10px;
				margin-right:10px;
				padding: 2px 5px 2px;
			}
			.article_menu>.s3{
				margin-left: 20px;
				margin-right:10px;
				padding: 2px 5px 2px;
			}
			.article_menu>.s4{
				margin-left: 30px;
				margin-right:10px;
				padding: 2px 5px 2px;
			}
            .article{
                margin: 5px;
            }
            .article>.title{
                margin: 5px 0 5px 5px;
                padding-left: 5px;
				border-bottom: 2px solid #c4c4c4;
            }
            .article>.article-body{
                margin: 5px 0 5px 5px;
                padding-left: 30px;
            }
            .article>.article-body h1{
				font-size: 2em;
                border-bottom: 2px solid #c4c4c4;
            }
            .article>.article-body h2{
				font-size: 1.8em;
                color: #626262;
            }
            .article>.article-body h3{
				font-size: 1.6em;
			}
            .article>.article-body h4{
				font-size: 1.4em;
			}
            .article>.article-body h5{
				font-size: 1.2em;
			}
        </style>
    </head>
    <body>
       <div class="container">
            <div class="navbar navbar-fixed-top">
                <div class="navbar-inner">
                <div class="container">
                    <a class="brand" href="../../article_list.html">CK Doc</a>
                     <ul class="nav">
                        <li><a href="../../article_list.html">Home</a></li>
                        <li class="active"><a href="../../article_list.html">Articles</a></li>
                        <li><a href="../../category.html">Category</a></li>
                        <li><a href="../../about.html">About me</a></li>
                     </ul>
                 </div>
                 </div>
            </div>
            <div id="main-body">
            <div class="row">
                <div class="span9">
                    <div class="article row">
                        <div class="title">
                            <h1 class="title-name">miRNA Analysis from omiRas</h1>
							<div class="row">
								<div class="span4"><i>2013-11-15 16:10:40</i></div>
							</div>
                        </div>
                        <div class="article-body">
							<h1><a name="1"></a>Thoughts</h1>

<p>今天下午看到另一個miRNA的differential expression分析的服務，叫做omiRas，稍微研究一下他的<a href="http://tools.genxpro.net/omiras/">Workflow</a>也跟我們的做一個比較。  </p>

<p><img src="http://i.imgur.com/TdW5zrt.png" alt="WorkFlow"></p>

<h1><a name="2"></a>omiRas</h1>

<h2><a name="3"></a>Simple Steps</h2>

<ul>
<li>Read pre-processing<br>
這部份其實跟miRDeep2做的事情是差不多的，把adapter拿掉b然後把重複的Sequence清一清，最後再拿Bowtie把資料mapping到genome上面，找mirdeep2的mapper.pl就可以做類似的事情。第4步是annotation及normalization，關於normalization我不熟，可能要再看paper一下。Quantification則是把mapping到非coding區域的reads拿給mirDeep2做De Novel預測，並且把所有表現的資料整理mapping的資訊及分佈，給下一步differential expression分析用<br></li>
<li>Combination of expression results and differential expression<br>
有表現的ncRNAs會被拿去DEseq做分析，產生differential expression的結果<br></li>
<li>Visualization
就是顯示結果</li>
</ul>

<p>與我目前的單位比較，其實大家做的事情勢差不多的，但是我們是直接將Differential expression及miRNA分析這兩塊獨立開來。用來Differential expression的分析工具我也不是很確定，但應該是tophat及cufflinks。</p>

<h2><a name="4"></a>Expression Result</h2>

<p><a href="http://tools.genxpro.net/omiras/0be30597ffef/results/expression/a/Wt_r1_fastq/">Expression Table</a><br>
<a href="http://tools.genxpro.net/omiras/0be30597ffef/results/plots/a/Wt_r1_fastq/">Mapping Statistics</a>  </p>

<p>Mappng Statistics會顯示RNA的區域(Intergenic, intronic, exonic, ncRNA)及ncRNA的種類比例(premiRNA, tRNA, rRNA, snoRNA, scRNA),這部份在我們的pipeline並沒有(雖然有mapping到genome的資訊,但是對於mapping annotation的表示我們就沒有再進一步顯示)。Sequence Length Distribution則是基本的檢驗read的依據,所以大家都一樣。Expression Table就差不多了,只是他們還有計算normalized<em>mapping</em>loci這項數據，我想應該是把read出現在不同loci的次數作normalization。</p>

<p>之前一直在想還需要再加上些什麼分析，現在看來其實做的事情都大同小異，只是怎樣把這些大家都需要的工具做良好的安排及包裝，方便大家使用。或許之後可以把differential expression及miRNA的pipeline結果合併顯示，不知道這樣會不會比較有幫助些。</p>

<h1><a name="5"></a>Refs</h1>

<p><a href="http://bioinformatics.oxfordjournals.org/content/early/2013/08/23/bioinformatics.btt457.abstract">Paper</a></p>

                        </div>
                    </div>
                </div>
				<div class="span3">
					<div class="article_menu">
						<div class="s1"><a href="#1">Thoughts</a></div><div class="s1"><a href="#2">omiRas</a></div><div class="s2"><i class='icon-chevron-right'></i><a href="#3">Simple Steps</a></div><div class="s2"><i class='icon-chevron-right'></i><a href="#4">Expression Result</a></div><div class="s1"><a href="#5">Refs</a></div>
					</div>
				</div>
			</div>
			<div class="row">
				<div class"span9">
			    	<div id="disqus_thread"></div>
				    <script type="text/javascript">
				        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
				        var disqus_shortname = 's4doc'; // required: replace example with your forum shortname

				        /* * * DON'T EDIT BELOW THIS LINE * * */
				        (function() {
				            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
				            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
				            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
						})();
					</script>
					<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript><a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
				</div>
			</div>
            </div>
       </div>
    </body>
</html>
